deepchecks/nlp/utils/text_properties.py

```diff
@@ -193,6 +193,38 @@ def formality(raw_text: Sequence[str], device: Optional[int] = None) -> List[flo
     classifier = get_transformer_pipeline('formality', model_name, device=device)
     return [x['score'] if x['label'] == 'formal' else 1 - x['score'] for x in classifier(raw_text)]
 
+def lexical_density(raw_text: Sequence[str]) -> List[str]:
+    """
+    Lexical density is the percentage of unique words in a given text. For more 
+    information: https://en.wikipedia.org/wiki/Lexical_density
+
+    Returns a list of floats of lexical density.
```

Comment:
```suggestion
    Return a list of floats of lexical density per text sample.
    
    Lexical density is the percentage of unique words in a given text. For more 
    information: https://en.wikipedia.org/wiki/Lexical_density
```

Reply:
Other wise it will fail on the code style CI test

Reply:
Done